recipes/laspy/meta.yaml

```diff
@@ -0,0 +1,70 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "laspy" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "9561d6d4cfb5ebfa8201151415d45085" %}
```

Comment:
When getting the source from pypi.io, you can get it from the page there:
https://pypi.org/project/laspy/#files

or generically, by following these instructions:

http://conda-forge.github.io/docs/meta.html#populating-the-hash-field